mod-configuration-server/src/main/java/org/folio/rest/impl/ConfigAPI.java

```diff
@@ -183,7 +181,7 @@ public void getConfigurationsEntriesByEntryId(String entryId, String lang, java.
         String tenantId = TenantTool.calculateTenantId( okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT) );
 
         Criterion c = new Criterion(
-          new Criteria().addField(idFieldName).setJSONB(false).setOperation("=").setValue("'"+entryId+"'"));
+          new Criteria().addField("id").setJSONB(false).setOperation("=").setVal(entryId));
```

Comment:
The change is correct, but the https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-25 suggests to replace the body of `getConfigurationsEntriesByEntryId` by a single invocation of PgUtil.getById.

Reply:
Yep.. But I was thinking that it would be more natural to take ALL at once and when making a PR for MODCONF-35 which I'll turn to after this.

Reply:
I mean: moving all to PgUtil when possible